arm/Microsoft.Authorization/policySetDefinitions/parameters/parameters.json

```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "policySetDefinitionName": {
+        "value": "test-policySetExample"
+      },
+      "policySetDescription": {
+        "value": "[Test] Set of security policies"
+      },
+      "displayName": {
+        "value": "[Test] contoso security Policies"
+      },
+      "policyDefinitionGroups": {
+        "value": [
+          {
+            "name": "Network"
+          },
+          {
+            "name": "ARM"
+          }
+        ]
+      },
+      "policyDefinitions": {
+        "value": [
+          {
+            "groupNames": [
+              "ARM"
+            ],
+            "parameters": {
+              "listOfAllowedLocations": {
+                "value": [
+                  "australiaeast"
+                ]
+              }
+            },
+            "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/e56962a6-4747-49cd-b67b-bf8b01975c4c",
+            "policyDefinitionReferenceId": "Allowed locations_1"
+          },
+          {
+            "groupNames": [
+              "ARM"
+            ],
+            "parameters": {
+              "listOfAllowedLocations": {
+                "value": [
+                  "australiaeast"
+                ]
+              }
+            },
+            "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/e765b5de-1225-4ba3-bd56-1ac6695af988",
+            "policyDefinitionReferenceId": "Allowed locations for resource groups_1"
+          },
+          {
+            "groupNames": [
+              "Network"
+            ],
+            "parameters": {
+              "effect": {
+                "value": "Audit"
+              }
+            },
+            "policyDefinitionId": "/providers/Microsoft.Management/managementGroups/mg-main/providers/Microsoft.Authorization/policyDefinitions/test-deny-keyvault-public-access",
+            "policyDefinitionReferenceId": "test-deny-keyvault-public-access"
+          }
+        ]
+      },
+      "metadata": {
+        "value": {
+          "category": "Security",
+          "version": "1"
+        }
+      },
+      "subscriptionId": {
+        "value": "20d6fbfe-b049-471c-95af-1369d14d0d45"
```

Comment:
tbr

Reply:
I'll change to the sub ID used by the service principal

Reply:
fixed